src/snowflake/snowpark/_internal/analyzer/snowflake_plan_node.py

```diff
@@ -236,6 +237,7 @@ def __init__(
         self.change_tracking = change_tracking
         self.copy_grants = copy_grants
         self.iceberg_config = iceberg_config
+        self.table_exists = table_exists
```

Comment:
add a comment here about what is this field used for

Reply:
added